errors/src/parser/parser_errors.rs

```diff
@@ -183,4 +183,12 @@ create_errors!(
         msg: "unable to parse array dimensions",
         help: None,
     }
+
+    /// For when the parser encountered a deprecated `mut self` parameter in a member function declaration.
+    @formatted
+    mut_self_parameter {
+        args: (),
+        msg: "`mut self` is deprecated. Use `&self` if you would like to pass in a mutable reference to `self`",
```

Comment:
I know this is used elsewhere, but "deprecated" suggests it can still be used although it is discouraged. At least that's what the Rust compiler means by the word. Here instead, you get an error.
```suggestion
        msg: "`mut self` is no longer accepted. Use `&self` if you would like to pass in a mutable reference to `self`",
```